scripts/test/Muon/FFTPresenter_test.py

```diff
@@ -0,0 +1,121 @@
+import mantid
+import mock
+#from . import Muon
+from  Muon import FFTPresenter 
+from  Muon import FFTView 
+import unittest
+import sys
+
+class FFTPresenterTest(unittest.TestCase):
+    def setUp(self):
+        self.view=mock.create_autospec(FFTView.FFTView,spec_set=True) 
+        #signals
+        self.view.tableClickSignal=mock.Mock(return_value=[1,3])
+        self.view.buttonSignal=mock.Mock()
```

Comment:
Good use of `mock`. I just have a couple of questions (out of personal interest). Whenever you have time it would be good to have a chat